src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgid "App Passwords"
+msgstr "App-wachtwoorden"
+
+#: src/view/screens/AppPasswords.tsx:232
+msgid "App password deleted"
+msgstr "App-wachtwoord verwijderd"
+
+#: src/view/screens/Settings/index.tsx:664
+msgid "App password settings"
+msgstr "App-wachtwoordinstellingen"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:148
+#: src/components/moderation/LabelsOnMeDialog.tsx:151
+msgid "Appeal"
+msgstr "Beroep"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:243
+msgid "Appeal \"{0}\" label"
+msgstr "Beroep tegen \"{0}\"-label"
+
+#: src/components/moderation/LabelsOnMeDialog.tsx:233
+#: src/screens/Messages/components/ChatDisabled.tsx:91
+msgid "Appeal submitted"
+msgstr "Beroep ingediend"
+
+#: src/screens/Messages/components/ChatDisabled.tsx:51
+#: src/screens/Messages/components/ChatDisabled.tsx:53
+#: src/screens/Messages/components/ChatDisabled.tsx:99
+#: src/screens/Messages/components/ChatDisabled.tsx:101
+msgid "Appeal this decision"
+msgstr "Beroep aantekenen tegen deze beslissing"
+
+#: src/screens/Settings/AppearanceSettings.tsx:89
+#: src/view/screens/Settings/index.tsx:485
+msgid "Appearance"
+msgstr "Weergave"
+
+#: src/Navigation.tsx:325
+msgid "Appearance Settings"
+msgstr "Weergave-instellingen"
+
+#: src/view/screens/Settings/index.tsx:476
+msgid "Appearance settings"
+msgstr "Weergave-instellingen"
+
+#: src/screens/Feeds/NoSavedFeedsOfAnyType.tsx:47
+#: src/screens/Home/NoFeedsPinned.tsx:93
+msgid "Apply default recommended feeds"
+msgstr "Standaard aanbevolen feeds toepassen"
+
+#: src/components/dialogs/MutedWords.tsx:433
+msgid "Are you sure?"
+msgstr "Weet u het zeker?"
+
+#: src/view/com/composer/Composer.tsx:532
+msgid "Are you sure you'd like to discard this draft?"
+msgstr "Weet u zeker dat u dit concept wilt laten vervallen?"
+
+#: src/view/screens/AppPasswords.tsx:283
+msgid "Are you sure you want to delete the app password \"{name}\"?"
+msgstr "Weet u zeker dat u het app-wachtwoord \"{name}\" wilt verwijderen?"
+
+#: src/components/dms/MessageMenu.tsx:149
+msgid "Are you sure you want to delete this message? The message will be deleted for you, but not for the other participant."
+msgstr "Weet u zeker dat u dit privébericht wilt verwijderen? Het privébericht wordt voor u verwijderd, maar niet voor de andere deelnemer."
+
+#: src/screens/StarterPack/StarterPackScreen.tsx:632
+msgid "Are you sure you want to delete this starter pack?"
+msgstr "Weet u zeker dat u dit startpakket wilt verwijderen?"
+
+#: src/screens/Profile/Header/EditProfileDialog.tsx:82
+msgid "Are you sure you want to discard your changes?"
+msgstr "Weet u zeker dat u uw wijzigingen ongedaan wilt maken?"
+
+#: src/components/dms/LeaveConvoPrompt.tsx:48
+msgid "Are you sure you want to leave this conversation? Your messages will be deleted for you, but not for the other participant."
+msgstr "Weet u zeker dat u dit gesprek wilt verlaten? Uw privéberichten worden voor u verwijderd, maar niet voor de andere deelnemer."
```

Comment:
It was assumed that "your messages" refers to messages in DM between two users. Therefore for consistency "priveberichten", because the English version use both "messages" and "direct messages" seemingly with the same meaning.

Do you happen to know the context? Are these "your messages" not DM?

Reply:
No, you are right. These are part of the `LeaveConvoPrompt`, inside dms.

Reply:
OK, thank you for checking against the context! My reading speed in TypeScript is not really high or more like non-existing. Closing this one.